src/ocamlorg_frontend/layouts/package_layout.eml

```diff
@@ -320,4 +320,4 @@ Layout.base
   }
 </script>
 <% | _ -> () ); %>
-<%s! Secondary_footer.render() %>
```

Comment:
This secondary footer still needs to be moved under the content area. Here is a page to test that it works: http://127.0.0.1:8080/p/irmin/latest/doc/index.html